src/services/userServices.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+const users = [];
```

Comment:
Tests are failing, because we are not clearinig this list.
After every start of the server in createServer try to clear both lists
`users` and `expenses`